GitHub/scripts/setup-runner.sh

```diff
@@ -42,16 +44,30 @@ case $1 in
     -l|--runner_labels)
     labels=$2
     ;;
+    -c|--cpu)
+    cpu=$2
+    ;;
+    -e|--ephemeral)
+    ephemeral=$2
+    ;;
 esac
 shift
 done
 
-mkdir $deploy_dir
+mkdir -p $deploy_dir
 
 curl -o $deploy_dir/actions-runner.tar.gz -L https://github.com/actions/runner/releases/download/v$version/actions-runner-osx-x64-$version.tar.gz
 cd $deploy_dir && tar xzf $deploy_dir/actions-runner.tar.gz
 
-$deploy_dir/config.sh --url $repository --token $github_token --name $runner --work $work_dir --runnergroup $group --labels $labels
+if [[ $ephemeral = "false" ]]
+then
+    $deploy_dir/config.sh --url $repository --token $github_token --name $runner --work $work_dir --runnergroup $group --labels $labels
```

Comment:
Since the command is largely the same, you can set it to a variable:

```sh
config_command="$deploy_dir/config.sh --url $repository --token $github_token --name $runner --work $work_dir --runnergroup $group --labels $labels"
```

And you can set the ephemeral flag as such:

```sh
elif [[ $ephemeral == "true" ]]; then
  config_command="$config_command --ephemeral"
else
```

And then finally execute the command after the checks:

```sh
eval $config_command
```